package.json

```diff
@@ -20,22 +20,22 @@
     "webpack-cli": "3.3.5"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap -- --no-package-lock && node scripts/reformat-with-prettier",
-    "bootstrap": "lerna bootstrap -- --no-package-lock && node scripts/reformat-with-prettier",
+    "postinstall": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier",
```

Comment:
Adding `--no-ci` to prevent running as `npm ci` which requires lock files or `npm-shrinkwrap.json` being present.